client/app/components/dashboards/ExpandWidgetDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import Modal from 'antd/lib/modal';
+import { VisualizationRenderer } from '@/visualizations/VisualizationRenderer';
+import { wrap as wrapDialog, DialogPropType } from '@/components/DialogWrapper';
+import { VisualizationName } from '@/visualizations/VisualizationName';
+
+function ExpandWidgetDialog({ dialog, widget }) {
+  return (
+    <Modal
+      {...dialog.props}
+      title={(
+        <React.Fragment>
+          <VisualizationName visualization={widget.visualization} />
+          <span>{widget.getQuery().name}</span>
+        </React.Fragment>
+      )}
+      width={900}
```

Comment:
@arikfr @gabrieldutra wdyt about going 100%?

Reply:
Probably makes sense, need to see it to get a feel.

Reply:
I like it with 900, it works as a `max-width`, so it goes 100% when using small resolutions and doesn't go that big on larger ones. In any case, I can update to 100% if you want to get the feel of it.

**900**
![Screenshot from 2019-08-18 13-11-23](https://user-images.githubusercontent.com/3356951/63227139-e7962c80-c1b9-11e9-8efc-d3bf20188c21.png)

**100%**
![Screenshot from 2019-08-18 13-11-51](https://user-images.githubusercontent.com/3356951/63227143-ed8c0d80-c1b9-11e9-9357-7a40dd857bbf.png)

Reply:
Maybe 95% or something that's a bit smaller than the current width of the page?

How does it decide on the height?